.github/filters.yaml

```diff
@@ -6,7 +6,7 @@ backend:
   - 'packages/**/strapi-server.js'
   - 'packages/{utils,generators,cli,providers}/**'
   - 'packages/core/*/{lib,bin,ee}/**'
-  - 'api-tests/**'
```

Comment:
@alexandrebodin I don't really understand what this file is doing, but does it actually need to include e2e, cli, and/or the entire tests dir?

Reply:
Short answer: currently adding the cli or e2e tests folder might just trigger api tests more often than necessary.

Long answer:
It's used to decide if certain jobs need to run or not in the github workflows. there is a filter action and we receive a boolean if files in this patterns were changed. 

That filter is curerntly only used to avoid running api tests. It might be worth using to avoid running cli or e2e tests if possible.